### DISCUSSION TITLE
Welcome to PiSphere-Project Discussions!

### DISCUSSION THREAD

**Discussion by KOSASIH** [KOSASIH/PiSphere-Project#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kianoclarissa** [comment]
Hallo bang senang bisa berhubungan dengan Abang..semangat melanjutkan projeknya ya bang semoga sukses

**Comment by MhmdApoAli** [comment]
Do you work with Pi Network officially?

**Reply by IsidroPT** [reply]
Lets go Pi , que visao , que projeto ...

**Comment by AlphaDZ47** [comment]
Good job 👍 🫂 💜 Thanks ⏩ ✈️

**Comment by tsotsoobi** [comment]
Awesome project, couldn't help but share it all over X, probably the biggest project we have come across on Blockchain
